### DISCUSSION TITLE
Week number is wrong

### DISCUSSION THREAD

**Discussion by SimasLitauer** [liamcain/obsidian-calendar-plugin#247] [General]
Showing wrong week number.
Now is week 18, but plugin showing 19
![Screenshot_211](https://user-images.githubusercontent.com/77072302/167245767-55fdd027-4dd2-4146-bfca-6a110e39cf83.png)

**Comment by fkuersch** [comment]
I also noticed that issue. For me, setting 'override locale' to 'de' fixed that

It has been discussed in the following issues:

Related #248 
Related #247 
Related #222 
Related #206 
Related #183 
Related #128 

Related liamcain/obsidian-periodic-notes#95
Related liamcain/obsidian-periodic-notes#55
Related liamcain/obsidian-periodic-notes#41
Related liamcain/obsidian-periodic-notes#17
Related liamcain/obsidian-periodic-notes#13

**Reply by craigpinegar** [reply]
Hi Liam. I also found this to be true. Every other calendar has 2022 week 1 starting on Sunday, January 2. Can you please align your calendar to it?
